.stubs/importlib_resources.pyi

```diff
@@ -0,0 +1 @@
+def __getattr__(name): ...  # type: ignore
```

Comment:
I actually pushed this accidentally to master and had to force push an undo. it is not ready.
can remove importlib commits from your stack and force push to the branch? if you are not sure how let's chat.

You may be able to just rebase on master, but I suspect my force push will cause issues.

Reply:
got it, i will clean up the branch then. thx for letting me know.